teerex/sgx-verify/src/lib.rs

```diff
@@ -378,21 +378,17 @@ pub struct SgxVerifiedReport {
 	pub build_mode: SgxBuildMode,
 }
 
-type SignatureAlgorithms = &'static [&'static webpki::SignatureAlgorithm];
+type SignatureAlgorithms = &'static [&'static dyn webpki::types::SignatureVerificationAlgorithm];
```

Comment:
Can you explain to me why this is dyn?

Reply:
Because `SignatureVerificationAlgorithm` is a trait? And `webpki::ring::RSA_PKCS1_2048_8192_SHA256` (and the alike) is a trait object by itself: `dyn SignatureVerificationAlgorithm`

Reply:
But it seems that this has changed in this case, can't we use an enum/struct like before? I will accept and merge the PR anyhow, I am just curious.

Reply:
but the `verify_for_usage` looks for something like this:
```
    pub fn verify_for_usage(
        &self,
        supported_sig_algs: &[&dyn SignatureVerificationAlgorithm],
```

Not sure if that's what you look for tho...

Reply:
Ah, I see, then I guess this was indeed an api change. Sorry for having been bothering you, while just being too lazy to dig in the dependency crate. Thanks a lot!

Reply:
All good!